src/rpcrawtransaction.cpp

```diff
@@ -833,6 +833,9 @@ UniValue sendrawtransaction(const UniValue& params, bool fHelp)
                 throw JSONRPCError(RPC_TRANSACTION_ERROR, state.GetRejectReason());
             }
         }
+        else  
```

Comment:
This needs to be inside an `#ifdef ENABLE_WALLET` pre-processor or building with wallet disabled gives a compile error.